core/src/autogluon/core/searcher/bayesopt/autogluon/searcher_factory.py

```diff
@@ -101,7 +102,7 @@ def _create_common_objects(**kwargs):
     model_args = GPModelArgs(
         num_fantasy_samples=kwargs['num_fantasy_samples'],
         random_seed=random_seed,
-        active_metric=DEFAULT_METRIC,
+        active_metric=kwargs.get('active_metric', DEFAULT_METRIC),
```

Comment:
Since active_metric is just an internal name (right?), we can just fix it to DEFAULT_METRIC, no?

Reply:
Done.